pkg/components/cert-manager/component.go

```diff
@@ -101,9 +101,6 @@ func (c *component) Metadata() components.Metadata {
 		Name: Name,
 		Namespace: k8sutil.Namespace{
 			Name: c.Namespace,
-			Labels: map[string]string{
```

Comment:
We could still replace it with new label, as it's defined in https://github.com/kinvolk/lokomotive/blob/0b0c39c449f1b916182539f987de357e284b87a9/assets/charts/components/cert-manager/templates/webhook-validating-webhook.yaml#L27, but given that it was not working anyway for such a long time, I guess it's fine to remove it :smile:

I recall I had issues with webhooks timing out on some old AKS version and I had to disable the webhooks to get this working, maybe that was the cause :smile: 

I doubt we have tests for it, so as long CI is passing, it should be fine to merge.

Reply:
guess I need to PR a cert-manager validator then ;) 
cert-manager used to create a certificate resource for itself which sometimes caused issued it now changed the way it signs it's own webhook (chicken egg issue there) for which the validating webhook not being ready should no longer cause issues

Reply:
on second thought better to double check as i saw somewhere you installed 2 clusterissuers also? Haven't taken a close look where exactly they are installed

Reply:
on third thought I just remembered that cluster issuers are of course cluster scoped (got confused with where the secrets are) :facepalm:

Reply:
Yeah, we install 2 clusterissuers as part of the cert-manager itself, so we support letsencrypt out of the box, both production and staging.